inputs/dust/dust_collision.in

```diff
@@ -0,0 +1,91 @@
+# ========================================================================================
```

Comment:
This is a pure drag input problem and so is out of scope for this PR unless I'm misunderstanding, so this input file and `pgen/dust_collision.hpp` should be removed.